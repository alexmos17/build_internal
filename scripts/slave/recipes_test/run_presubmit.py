# Copyright (c) 2013 The Chromium Authors. All rights reserved.
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.

"""Test inputs for recipes/run_presubmit.py"""

def blink_bare_test(api):
  return {
    'factory_properties': {'repo_name': 'blink_bare'},
    'build_properties': api.tryserver_build_properties(
      root='src/third_party/WebKit'
    ),
  }

def blink_test(api):
  return {
    'factory_properties': {'repo_name': 'blink'},
    'build_properties': api.tryserver_build_properties(
      root='src/third_party/WebKit'
    ),
  }

def tools_build_test(api):
  return {
    'factory_properties': {'repo_name': 'tools_build'},
    'build_properties': api.tryserver_build_properties(
      root='src'
    ),
  }

def chromium_test(api):
  return {
    'factory_properties': {'repo_name': 'chromium'},
    'build_properties': api.tryserver_build_properties(
      root='src'
    ),
  }