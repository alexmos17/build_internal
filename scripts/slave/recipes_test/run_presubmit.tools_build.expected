[
  {
   "cmd": ["[DEPOT_TOOLS_ROOT]/gclient", "config", "--spec", "solutions = [{'url': 'https://chromium.googlesource.com/chromium/tools/build.git', 'managed': True, 'name': 'build', 'deps_file': '.DEPS.git'}]\n"],
   "name": "gclient setup",
   "seed_steps": ["gclient setup", "gclient clean", "gclient sync"]
  },
  {
   "cmd": ["[DEPOT_TOOLS_ROOT]/gclient", "revert", "--nohooks"],
   "name": "gclient clean"
  },
  {
   "cmd": ["[DEPOT_TOOLS_ROOT]/gclient", "sync", "--nohooks"],
   "name": "gclient sync"
  },
  {
   "cmd": ["git", "--work-tree", "[SLAVE_BUILD_ROOT]/build", "--git-dir", "[SLAVE_BUILD_ROOT]/build/.git", "config", "user.email", "commit-bot@chromium.org"],
   "name": "git config user.email",
   "seed_steps": ["git config user.email", "git config user.name", "git clean"]
  },
  {
   "cmd": ["git", "--work-tree", "[SLAVE_BUILD_ROOT]/build", "--git-dir", "[SLAVE_BUILD_ROOT]/build/.git", "config", "user.name", "The Commit Bot"],
   "name": "git config user.name"
  },
  {
   "cmd": ["git", "--work-tree", "[SLAVE_BUILD_ROOT]/build", "--git-dir", "[SLAVE_BUILD_ROOT]/build/.git", "clean", "-xfq"],
   "name": "git clean"
  },
  {
   "cmd": ["[DEPOT_TOOLS_ROOT]/apply_issue", "-r", "[SLAVE_BUILD_ROOT]/build/", "-i", 12853011, "-p", 1, "-s", "https://chromiumcodereview.appspot.com", "-e", "commit-bot@chromium.org"],
   "name": "apply_issue"
  },
  {
   "cmd": ["[DEPOT_TOOLS_ROOT]/presubmit_support.py", "--root", "[SLAVE_BUILD_ROOT]/build/", "--commit", "--verbose", "--verbose", "--issue", 12853011, "--patchset", 1, "--skip_canned", "CheckRietveldTryJobExecution", "--skip_canned", "CheckTreeIsOpen", "--skip_canned", "CheckBuildbotPendingBuilds", "--rietveld_url", "https://chromiumcodereview.appspot.com", "--rietveld_email", "", "--rietveld_fetch"],
   "name": "presubmit"
  }
]